tools/agent-ctl/src/main.rs

```diff
@@ -227,11 +204,11 @@ fn real_main() -> Result<()> {
                     .help("Allow interactive client"),
                     )
                 .arg(
-                    Arg::with_name("vsock-port")
-                    .long("vsock-port")
-                    .help("VSOCK Port number")
+                    Arg::with_name("server-address")
+                    .long("server-address")
+                    .help("server URI (vsock:// or unix://")
```

Comment:
Hi @jodh-intel, here is a typo in line 209 , missing a ')'  before the last "

Reply:
Good catch @lining2020! Fixed :smile: